chutney/server/src/main/java/com/chutneytesting/execution/domain/campaign/CampaignExecutionEngine.java

```diff
@@ -238,7 +239,7 @@ private Consumer<TestCaseDataset> executeScenarioInCampaign(Campaign campaign, C
             // Is stop requested ?
             if (!currentCampaignExecutionsStopRequests.get(campaignExecution.executionId)) {
                 // Init scenario execution in campaign report
-                campaignExecution.startScenarioExecution(testCaseDataset, campaign.executionEnvironment(), campaignExecution.userId);
+                campaignExecution.startScenarioExecution(testCaseDataset, campaign.executionEnvironment(), campaignExecution.userId, campaign.tags);
```

Comment:
the tags does not change betwen init and start execution, we can keep only the first one